Sprint-3/implement/get-angle-type.js

```diff
@@ -25,3 +25,21 @@
 // Identify Reflex Angles:
 // When the angle is greater than 180 degrees and less than 360 degrees,
 // Then the function should return "Reflex angle"
+
+function getAngleType(angleDegree) {
```

Comment:
Nice! Take a moment to refactor this. Could you make this simpler? What would happen if you just made the if statements and didn't bother with the else?

Reply:
Thank you for the suggestion
I've refactored the code as per your advice and pushed the changes. Let me know if there's anything else to improve.